CODE_OF_CONDUCT.md

```diff
@@ -5,33 +5,17 @@ See commit history to see our changes.
 ## Our Pledge
 
 We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, caste, color, religion, or sexual
-identity and orientation.
-
-We pledge to act and interact in ways that contribute to an open, welcoming,
-diverse, inclusive, and healthy community.
+community a harassment-free experience for everyone.
 
 ## Our Standards
 
-Examples of behavior that contributes to a positive environment for our
-community include:
-
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the overall
-  community
-
```

Comment:
I kinda liked this section, it may seem trivial, but it helps understanding the kind of community we aim to have. Explicitly stating the expected positive behavior helps reinforcing what we value as a community, and may help new people get in the community.

As as example, I have kinda severe social anxiety, and I was pretty scared of entering the discord at first (I didn't even do so before my first PR was merged and I started doing my second one, and even then I took a long while to actually start interacting in it xd). Reading these points helped me a lot in trusting that nothing would go wrong once I got in, that people would be nice and I wouldn't be hurt by someone being overly aggressive and stuff (according to my own metrics of course, things that are offensive to some may seem fine for others).

Having only examples of unacceptable behavior leaves out a lot of possible behavior that may seem OK for some, but not so for others. Examples of expected behavior help set up expectations and mitigate such "on the edge" behavior. Saying that empathy and kindness towards other people is expected explicitly is much more impactful and welcoming than saying nothing, even though it may be implicitly implied for you (of course we aim to have a kind and cool community, but saying it out loud helps a lot, both in shaping the community, and in having new people know what to expect). 

As such, in my opinion this section served a purpose, and should be left in!

Reply:
Idk it just read as pretty condescending to me

Reply:
Making these kinds of things overly vague is how you end up with assholes toeing the line. And trust me, there *are* people out there who will do things like that, even if they may not be in the community now.

Reply:
This doesn't make it vague though it just makes it more concise in telling people what they're not allowed to do and what sort of punishments they'll get.